src/Akihabara/Native/UnsafeNativeMethods.cs

```diff
@@ -30,22 +30,22 @@ public partial class UnsafeNativeMethods : NativeMethods
 
         #region GoogleLogging
 
-        [DllImport(MediaPipeLibrary, ExactSpelling = true, CharSet = CharSet.Unicode)]
+        [DllImport(MediaPipeLibrary, ExactSpelling = true)]
         public static extern MpReturnCode google_InitGoogleLogging__PKc(string name, string logDir);
 
         [DllImport(MediaPipeLibrary, ExactSpelling = true)]
         public static extern MpReturnCode google_ShutdownGoogleLogging();
 
-        [DllImport(MediaPipeLibrary, ExactSpelling = true, CharSet = CharSet.Unicode)]
+        [DllImport(MediaPipeLibrary, ExactSpelling = true)]
         public static extern MpReturnCode glog_LOG_INFO__PKc(string str);
 
-        [DllImport(MediaPipeLibrary, ExactSpelling = true, CharSet = CharSet.Unicode)]
+        [DllImport(MediaPipeLibrary, ExactSpelling = true)]
         public static extern MpReturnCode glog_LOG_WARNING__PKc(string str);
 
-        [DllImport(MediaPipeLibrary, ExactSpelling = true, CharSet = CharSet.Unicode)]
+        [DllImport(MediaPipeLibrary, ExactSpelling = true)]
```

Comment:
Are we sure removing charset is safe? this might bring more problems than benefits.

Reply:
I make most modifications by seeing if we didn't exactly match Homuler's current code in logic.
Are we really sure that adding CharSet is a good idea here?

Reply:
Not sure the rationale with it but @homuler can confirm why we need it in Unicode.

Reply:
This is related to https://github.com/vignetteapp/Akihabara/pull/31#issue-743420577
If you need to marshal Unicode strings here, you need to specify `Charset = Charset.Unicode`, but in the original code, it's assumed that only ANSI string will be given to this API.
https://github.com/vignetteapp/Akihabara/blob/366a799c98138ea4564f15829dc10a042479b9f6/src/Akihabara/Core/MpResourceHandle.cs#L78

Reply:
I checked at hand, and it seems that even when passing unicode, it may be correct to not specify `Charset.Unicode`.

Reply:
Awesome! thanks for confirming.